### PR TITLE
feat: frontend updated readme

### PR DESCRIPTION
**Pull request by infonl-marcel**
frontend updated readme

Solves: PZ-4269)

### CODE REVIEW

File: src/main/app/README.md [hunk 1]
```diff
@@ -2,25 +2,29 @@
...
 
-Run `ng build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build.
+Run `npm run build` to build the project. The build artifacts will be stored in the `dist/` directory. Use the `--prod` flag for a production build. Run `npm run start` to run the built version.
 
 ## Running unit tests
 
-Run `ng test` to execute the unit tests via [Jest](https://jestjs.io/).
+Run `npm run test` or `npm run test:watch` to execute the unit tests via [Jest](https://jestjs.io/).
 
 ## Running end-to-end tests
```
Comment (edgarvonk) [inline src/main/app/README.md:25]: I would remove this entire section. We already have docs on how to run the e2e tests. Have nothing to do with this frontend Angular folder anyway.

File: src/main/app/README.md [hunk 2]
```diff
@@ -2,25 +2,29 @@
 
 This project was generated with [Angular CLI](https://github.com/angular/angular-cli) version 9.0.7.
```
Comment (edgarvonk) [inline src/main/app/README.md:3]: Is this still relevant?

File: src/main/app/README.md [hunk 3]
```diff
@@ -2,25 +2,29 @@
```
Comment (edgarvonk) [inline src/main/app/README.md:2]: I would change the title from `App` to e.g. `Zaakafhandelcomponent frontend application` ?

**Comment by infonl-marcel** [comment]
> Our workflow is to always create a JIRA issue (task) and then refer to that in our PRs. No PR without a JIRA ticket. That is how we roll. :-)

Added https://dimpact.atlassian.net/browse/PZ-4269

File: src/main/app/README.md (hunk 2 above)
Comment (infonl-marcel) [reply src/main/app/README.md:3]: Yes and No; could be of interest to have an idea about the Webpack bundler etc stuff if at any time there is some things happen. Of course there is the abundance of packages. I personally do like to know where the origine is coming from.

File: src/main/app/README.md (hunk 1 above)
Comment (infonl-marcel) [reply src/main/app/README.md]: Removed

File: src/main/app/README.md (hunk 3 above)
Comment (infonl-marcel) [reply src/main/app/README.md:2]: Adjusted

**Review by edgarvonk — APPROVED** [review]

**Review by bas-info-nl — APPROVED** [review]

**Pull request merged**